### PR TITLE
Uuid

### PR DESCRIPTION
**Pull request by dwightguth**
@edgar-pek please review

Basically adding a function to K that returns UUIDs

### CODE REVIEW

File: k-distribution/include/ocaml/prelude.ml [hunk 1]
```diff
@@ -217,6 +216,8 @@ let get_exit_code(subst: k Subst.t) : int = match (Subst.fold (fun k v res -> ma
   | [Int i], Some _ -> failwith "Bad exit code pattern"
   | _ -> res) subst None) with Some i -> i | _ -> failwith "Bad exit code pattern"
 
+let () = Random.self_init ()
```
Comment (edgar-pek) [inline k-distribution/include/ocaml/prelude.ml]: How is this related to UIUD?

Comment (dwightguth) [inline k-distribution/include/ocaml/prelude.ml]: it was needed in an earlier version of this diff and I forgot to take it out.

**Comment by edgar-pek** [comment]
got it. looks good.
